ntc_templates/templates/cisco_ios_show_power_inline.textfsm

```diff
@@ -21,4 +21,4 @@ Interfaces
   ^\s+\(Watts\)
   ^-+
   ^\s*$$
-  ^. -> Error
+  ^.

```

Comment:
There must be an [error directive](https://github.com/networktocode/ntc-templates/blob/master/docs/dev/dev_parser.md#error-directive).

I'd be glad to assist with updating the template if you provide example test data.
Thank you!

```suggestion
  ^. -> Error
```